src/libcore/hash/sip.rs

```diff
@@ -120,6 +121,7 @@ impl SipHasher {
         self.ntail = 0;
     }
 
+    #[inline]
     fn write(&mut self, msg: &[u8]) {
```

Comment:
Yeah, but that's also the most important function to have available for inlining, because most of it is thrown away when the compiler sees that, for example, it's only called with once with a single 8 byte slice. See my remark below about `#[inline(Maybe)].